extra/1-radio-stations.js

```diff
@@ -14,6 +14,11 @@
  */
 
 // `getAllFrequencies` goes here
+ function getAllFrequencies() {
+  let freqArr = [87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108];
+  return freqArr;
+ }
```

Comment:
this works, but to avoid having to type out all the numbers you can use a for loop here